observability/opentelemetry-metrics-backend/src/test/scala/sttp/client3/opentelemetry/OpenTelemetryMetricsBackendTest.scala

```diff
@@ -182,6 +182,65 @@ class OpenTelemetryMetricsBackendTest extends AnyFlatSpec with Matchers with Opt
     getHistogramValue(reader, OpenTelemetryMetricsBackend.DefaultLatencyHistogramName).map(_.getSum) should not be empty
   }
 
+  it should "use error counter when http error is thrown" in {
+    // given
+    val backendStub = SttpBackendStub.synchronous.whenAnyRequest.thenRespondServerError()
+    val eitherBackend = new EitherBackend(backendStub)
```

Comment:
hm do we need the either backend - or rather, is the wrapping correct? If we do `OpenTelemetryMetricsBackend(EitherBackend(stub))`, all exceptions are converted into `Eithers` before they reach metrics?

Reply:
Fixed